rt_gene_model_training/pytorch/utils/LearningRateFinder.py

```diff
@@ -0,0 +1,111 @@
+import math
+import os
+
+import h5py
+import matplotlib.pyplot as plt
+import torch
+from torch.utils.data import DataLoader
+from torch.utils.tensorboard import SummaryWriter
+from tqdm import trange
+
+from gaze_estimation_models_pytorch import GazeEstimationmodelResnet18
+from rtgene_dataset import RTGENEH5Dataset
+
+
+class RTGENELearningRateFinder(object):
+
+    def __init__(self, model, optimiser, loss, batch_size=128):
+        self._device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        self.writer = SummaryWriter()
+
+        data_train = RTGENEH5Dataset(h5_file=h5py.File(os.path.abspath("../../../RT_GENE/dataset.hdf5"), mode="r"), subject_list=list(range(16)))
```

Comment:
Would be nice to use `root_dir` here, too

Reply:
done.